addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -331,7 +363,7 @@ return core.Class.extend({
             if (typeof tip.run === "function") {
                 tip.run.call(tip.widget, action_helper);
             } else if (tip.run !== undefined) {
-                var m = tip.run.match(/^(click|text|drag_and_drop) *(?:\(? *["']?(.+)["']? *\)?)?$/);
+                var m = tip.run.match(/^([a-zA-Z0-9_]+) *(?:\(? *(.+?) *\)?)?$/);
```

Comment:
Oh I did not know about the .+?, maybe this is worth to be in a separate commit as well as a fix.